package NKikimrSysView;

option java_package = "ru.yandex.kikimr.proto";

enum ESysViewType {
    EPartitionStats = 1;
}
